app/controllers/admin/articles_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Admin
+  class ArticlesController < ApplicationController
+    def index
+      @articles = Article.all.order("title ASC").page params[:page]
+    end
+
+    def edit
+      @article = Article.find(params[:id])
+    end
+
+    def update
+      @article = Article.find(params[:id])
+
+      # if @article.update(article_params)
+      #   redirect_to admin_article_path(@article), notice: "Article Updated Successfully"
+      # else
+      #   render edit_admin_article_path(@article)
+      # end
+
+      respond_to do |format|
+        if @article.update_attributes(article_params)
+          format.html { redirect_to(admin_article_path(@article), notice: 'Article was successfully updated.') }
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.